package.json

```diff
@@ -4,7 +4,7 @@
   "description": "library to help create a reports service",
   "main": "cli.js",
   "scripts": {
-    "test": "eslint . && tap --cov test"
+    "test": "eslint . && NODE_ENV=test tap --cov test/addReport.test.js"
```

Comment:
why just addReport?

Reply:
It reports a 'skip' for the files in test/report if I do the whole dir, which was annoying me.  addReport.test.js is the only file with tests in it but can change this to *.test.js if you prefer.

Reply:
yeah, lets do *.test.js